napi/src/options.rs

```diff
@@ -23,14 +23,16 @@ pub struct NapiResolveOptions {
     /// Create aliases to import or require certain modules more easily.
     /// A trailing $ can also be added to the given object's keys to signify an exact match.
     ///
+    // FIXME can be array
```

Comment:
Let's have an issue instead of leaving a FIXME here without context. I'll clean this up.